npm/azure-npm.yaml

```diff
@@ -158,11 +159,10 @@ data:
         "MaxPendingNetPols":            100,
         "Toggles": {
             "EnablePrometheusMetrics": true,
-            "EnablePprof":             true,
+            "EnablePprof":             false,
```

Comment:
oh sorry, could we actually leave this on? Might help with debugging

Reply:
Updated!